_pages/ato/checklist.md

```diff
@@ -24,7 +24,6 @@ To start the security clearance process, [create an issue in the DevOps reposito
 
 ### Project team
 
-1. [ ] Initiate [ATO delegation](https://pages.18f.gov/before-you-ship/ato/delegation/)
```

Comment:
@NoahKunin Just for my own learning, can you give some insight about why this was removed?